python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -1308,8 +1420,7 @@ def _initialize_adapter(self) -> BaseAdapter:
             register_adapter(config, get_mp_context())
 
         adapter = cast(BaseAdapter, get_adapter(config))
-        # reset the adapter since the dummy flags may be different from the flags for the actual subcommand
-        reset_adapters()
```

Comment:
Did we confirm that we can remove this with no reprecussions?

Reply:
I spoke to Phil about it - the reason it was included was to avoid affecting the actual dbt execution, but that execution is isolated to a separate process, so I don't think it will actually have any effect. The implementation of `AdapterContainer` doesn't have any x-process/shared memory attributes, so there shouldn't be any meaningful bleed.